pages/glossary/distinguishable-box-shadows.md

```diff
@@ -10,4 +10,4 @@ Look at the [computed](https://drafts.csswg.org/css-cascade/#computed-value) val
 Two elements have a _distinguishable box-shadow_ if all of the following are true:
 
 - the [computed](https://drafts.csswg.org/css-cascade/#computed-value) values of their `box-shadow` property are different; and
-- if the difference is just based on the [color](https://www.w3.org/TR/css-backgrounds-3/#shadow-color) values, and only one of the elements has the color value specified then that value has to be different from the [computed](https://drafts.csswg.org/css-cascade/#computed-value) `background-color` of the element and the value's [alpha component](https://drafts.csswg.org/css-color/#alpha-channel) has to be different from 0.
+- if the difference is just based on the [color](https://www.w3.org/TR/css-backgrounds-3/#shadow-color) values, and only one of the elements has the color value specified then that value has to be different from the [computed](https://drafts.csswg.org/css-cascade/#computed-value) `background-color` of the element and the value's [alpha component](https://drafts.csswg.org/css-color/#alpha-channel) has to be positive.
```

Comment:
Alpha channel actually ranges from 0 to 1, so "different from 0" is the same as "positive" 😀 
(I tend to always be confused between "positive" and "non-negative" because we don't have that separation in French)

(aka, this is correct and no need to change it)

Reply:
It very much depends on contextual convention whether or not you consider zero as non-positive though so the current phrasing seems ambiguous.

Reply:
But the actual meaning of the word "positive" (resp. "negative") in English is "strictly greater than zero" (resp. "strictly lower than zero"). As can be checked in any English dictionary. In English, zero is **neither positive nor negative** (and thus both "non-positive" and "non-negative"…) While the sign bit can be set or not on 0 (and in some cases, both +0 and -0 are valid representations), the `sign` function is often ternary. See also https://en.wikipedia.org/wiki/Sign_bit which takes care in not saying that the sign bit indicate whether a number is positive or negative (which it indeed doesn't).

Native English speakers tend to know that pretty well (from my experience of writing mathematical texts)… And feel surprised when us non-native speakers are confused at the difference between "positive" and "non-negative"… (and try to use "strictly positive" which is useless in English).

So, I do feel the same as you that "positive" is confusing, but actually it is not and only has one precise mathematical meaning in English…
Of course, since it feels confusing to many, we still might want to use another wording…

Reply:
The dictionary definition of "positive" doesn't really help us much here as the real world is nasty, [signed zero](https://en.wikipedia.org/wiki/Signed_zero) is a thing, and JavaScript, the lingua franca of the web, uses IEEE 754 floating point numbers: https://github.com/MikeMcl/decimal.js/issues/62#issuecomment-317803356. This StackOverflow answer, the top result on Google for the query "javascript is positive", makes the same misstep: https://stackoverflow.com/a/27295034

Reply:
Changed to "greater than zero". That should not be ambiguous!